lib/validate.js

```diff
@@ -11,17 +11,21 @@ function validate(
     includeKeys = false,
     traceErrorMessage = 'Data did not match schema',
     traceSuccessMessage = 'Data after validation',
+    allowAndStripUnknownKeys = true,
```

Comment:
i have no clue what all this is used for, but my first thought here was whether any consumers of this function would ever need different values for these two keys (allowUnknown false and stripUnknown true)?

Reply:
It seems unsafe to `allowUnknown` and not `stripUnknown` so if we did that, I'd tend to think we weren't doing a good job of validating.

Originally I'd passed `options` directly through which would have supported any combination, though I'm not sure we would ever want to do that.